docs/DafnyRef/Modules.md

```diff
@@ -340,7 +340,8 @@ module X {
 }
 ```
 `Option.a` now means the `a` in the datatype instead of the `a` in the module.
-To avoid confusion in such cases, it is an ambiguity error to `import open`
+To avoid confusion in such cases, it is an ambiguity error to use such a name
```

Comment:
I find this sentence a bit hard to parse.

Reply:
It is a complex corner case. I revised.